### DISCUSSION TITLE
Is this mouse movement behaviour between monitors a feature or bug?

### DISCUSSION THREAD

**Discussion by maltbeverage** [hyprwm/Hyprland#6355] [General]
I noticed that if I configure my monitors with a gap in the coordinates as shown:

`monitor=desc:$monitorLeft,1920x1080@60,-100x-50,1`
`monitor=desc:$monitorCenter,2560x1440@144,1920x0,1`

the mouse will not transition between the monitors unless it is moved at a high speed, almost a flicking motion. 

I discovered this by accident and find it quite useful to isolate my left most monitor from accidental console input redirection into chat. 

I've spent a while combing the wiki but can't find anything about the behavior. I'm curious is this is an intentional feature or just a byproduct of how the monitor configuration works.

**Comment by vaxerski** [comment]
bug (it should not be able to go through regardless of speed)

**Reply by maltbeverage** [reply]
Thanks for the info. Can follow_mouse behavior be overridden with window rules? For example have follow_mouse=1 in the config but for a specific application, have it act like follow_mouse=2.
